python/ray/serve/_private/controller.py

```diff
@@ -172,12 +173,14 @@ async def __init__(
             if actor["namespace"] == SERVE_NAMESPACE
         ]
 
+        self.autoscaling_state_manager = AutoscalingStateManager()
```

Comment:
Is checkpointing this feasible/desirable? It would be nice if the controller could recover with the existing autoscaling information now that everything is centralized.

No need to do it in this PR, but if it's doable, it would be a nice follow-up.